constants/styles/MainScreen.ts

```diff
@@ -102,7 +103,7 @@ export const styles = (isDark: boolean) => StyleSheet.create({
         justifyContent: "center"
     },
     addStyledButton: {
-        marginTop: PixelRatio.getPixelSizeForLayoutSize(7.5) * -1,
+        marginVertical: PixelRatio.getPixelSizeForLayoutSize(7.5) * -1,
```

Comment:
We do remove spacing under the button with this modification. As between the photo browser and the header there should be space between here and the tab navigator as well, you think?

Reply:
Yes I'll change this

Reply:
Thanks.